build/TestShared/GlobalSuppressions.cs

```diff
@@ -5,17 +5,3 @@
 // attributes that are applied to this project.
 // Project-level suppressions either have no target or are given 
 // a specific target and scoped to a namespace, type, member, etc.
```

Comment:
This file is now effectively empty (it only has comments). You should delete it.

Reply:
Makes sense, we should delete this.